src/configuration/routes.md

```diff
@@ -115,6 +115,85 @@ https://blog.example.com.sprint-7onpvba-tvh56f275i3um.eu-2.platformsh.site/
 
 If your project involves only a single apex domain with one app or multiple apps under subdomains, it's generally best to use the `{default}` placeholder.  If you are running [multiple applications](/configuration/app/multi-app.md) on different apex domains then you will need to use a static domain for all but one of them.
 
+## Route identifiers
+
+All routes defined on an environment will be available in the application in the `PLATFORM_ROUTES` environment variable.  That value is a base64-encoded JSON string, which can be extracted in the language of your choice to look up the effective routes that were generated.
```

Comment:
- "available in the application" -> "available to the application"
- "extracted in the language of your choice" -> "extracted with the language of your choice"
- "look up the effective routes that were generated" -> ???

Reply:
I'm not sure what that last one actually means. I mean, it's explained in the next sentence, but the wording is really awkward.

Reply:
It is a but clunky.  Suggestions for better wording welcome.